extensions/podman/packages/extension/src/checks/windows/wsl2-check.ts

```diff
@@ -54,14 +53,11 @@ export class WSL2Check extends MemoizedBaseCheck {
     }
   }
 
-  async isUserAdmin(): Promise<boolean> {
-    const client = await getPowerShellClient(this.telemetryLogger);
-    return client.isUserAdmin();
-  }
-
   async executeImpl(): Promise<CheckResult> {
     try {
-      const isAdmin = await this.isUserAdmin();
+      const userAdminResult = await this.userAdminCheck.execute();
+      const isAdmin = userAdminResult.successful;
+
```

Comment:
suggestion: it seems we need isAdmin only if !isWsl, so shouldn't be initialized/used only if isWsl is not ok ?

Reply:
https://github.com/podman-desktop/podman-desktop/pull/14727